demo/jest.preprocessor.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+/* eslint-env node */
+
+"use strict";
+
```

Comment:
Should we create an issue and/or add a TODO in the code to remind us of the conditions under which we can remove this file?

Reply:
Yes, sounds good.

Reply:
Done https://github.com/tasitlabs/tasit/issues/31